src/main/java/seedu/address/logic/commands/DownCommand.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.ItemModel;
+
+/**
+ * Command for scrolling down.
+ */
+public class DownCommand extends Command {
+
+    public static final String COMMAND_WORD = "down";
+    public static final String MESSAGE_SUCCESS = "Scrolling down...";
+    public static final String MESSAGE_USAGE = "Message Usage";
+
+    private String pane;
+
+    public DownCommand(String input) {
+        switch(input.trim()) {
+        case "L":
```

Comment:
Do you think it would make it more user-friendly to allow both `L` and `l`?

Reply:
Done.